presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestingPostgreSqlServer.java

```diff
@@ -32,14 +31,15 @@
     private static final String DATABASE = "tpch";
     private static final int POSTGRESQL_PORT = 5432;
 
-    private final DockerContainer dockerContainer = DockerContainer.forImage("postgres:10.3")
-            .setPorts(POSTGRESQL_PORT)
-            .setEnvironment(ImmutableMap.of(
-                    "POSTGRES_PASSWORD", PASSWORD,
-                    "POSTGRES_USER", USER,
-                    "POSTGRES_DB", DATABASE))
-            .setHealthCheck(docker -> execute(getJdbcUrl(docker), "SELECT 1"))
```

Comment:
proper health check is not yet there: https://github.com/testcontainers/testcontainers-java/issues/1529

Reply:
so is this a draft PR?

Reply:
You should be able to wait for the port to be available, or do a custom check query.

Reply:
Isn't this default behaviour? https://www.testcontainers.org/features/startup_and_waits/

Reply:
I was thinking about running a `SELECT 1`. 

There is also a way to run some custom command to see if psql is up and running, see: https://github.com/testcontainers/testcontainers-java/issues/1529#issuecomment-559560210

Reply:
FYI Testcontainers' PostgreSQL module runs `SELECT 1` as a check (see wait strategies)

health check is Docker's mechanism and is not very useful for us (Testcontainers), our check also verifies the connectivity between the test host and container (and verifies that the JDBC driver is valid)

Reply:
Also, is there a reason you're not using the PostgreSQL module? `org.testcontainers:postgresql`?

Reply:
Does `org.testcontainers:postgresql` pull postgres JDBC as a dep?

Reply:
Also FYI:
you can start containers via JDBC URL:
https://www.testcontainers.org/modules/databases/#database-containers-launched-via-jdbc-url-scheme

Reply:
> Does org.testcontainers:postgresql pull postgres JDBC as a dep?

No, you need to bring your own driver